installer/roles/create-mobile-client-crd/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+# tasks file for create-mobile-client-crd
+
+- name: Copy CRD definition to tmp file
```

Comment:
Small enough thing. Could we just use `copy` here, I don't see any variable substitutions being made in `mobile-client-crd.yaml`.

http://docs.ansible.com/ansible/latest/copy_module.html

Reply:
True, I'm just used to using template, i'll switch it.